src/components/Payroll/PayrollHistory/PayrollHistoryPresentation.tsx

```diff
@@ -0,0 +1,168 @@
+import { useTranslation } from 'react-i18next'
+import styles from './PayrollHistoryPresentation.module.scss'
+import { DataView, Flex } from '@/components/Common'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useI18n } from '@/i18n'
+import { HamburgerMenu } from '@/components/Common/HamburgerMenu'
+import { formatNumberAsCurrency } from '@/helpers/formattedStrings'
+import ListIcon from '@/assets/icons/list.svg?react'
+import TrashcanIcon from '@/assets/icons/trashcan.svg?react'
+
+export interface PayrollHistoryItem {
+  id: string
+  payPeriod: string
+  type: 'Regular' | 'Off-cycle' | 'Dismissal'
```

Comment:
you can use this type from my PR if you want
https://github.com/Gusto/embedded-react-sdk/pull/570